packages/cli/commons/src/utils/ux.ts

```diff
@@ -0,0 +1,16 @@
+import {CliUx} from '@oclif/core';
+import {red, green} from 'chalk';
+
+export function starSpinner(task: string) {
+  if (CliUx.ux.action.running) {
+    CliUx.ux.action.stop(green('✔'));
```

Comment:
Ain't the green check mark a bit optimisitic?
What make you sure that if we start another spinner the previous was successful?

Reply:
Unless you intentionally wrap an async task (along with a spinner) within a try-catch block so you can continue with the execution, any caught error by the base command will stop the spinner with a failure.

I guess, if you manually prevent the command class from handling the error, you should also handle the spinner behaviour with `stopSpinner(err)`